client/src/endpoint/account.rs

```diff
@@ -452,3 +452,180 @@ mod effects_tests {
         assert_eq!(req.uri().query(), Some("cursor=CURSOR&order=asc&limit=123"));
     }
 }
+
+/// An endpoint that accesses the Payment operations where a specific account is either the
+/// sender or receiver.
+#[derive(Debug, Clone)]
+pub struct Payments {
+    id: String,
+    cursor: Option<String>,
+    order: Option<Order>,
+    limit: Option<u32>,
+}
+
+impl Payments {
+    /// Returns a new endpoint for account payments. Hand this to the client in order to request
+    /// payment operations for a specific account.
+    pub fn new(id: &str) -> Self {
+        Self {
+            id: id.to_string(),
+            cursor: None,
+            order: None,
+            limit: None,
+        }
+    }
+
+    /// Starts the page of results at a given cursor
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// use stellar_client::sync::Client;
+    /// use stellar_client::endpoint::account;
+    /// # use stellar_client::endpoint::transaction;
+    /// # use stellar_client::endpoint::Order;
+    ///
+    /// let client   = Client::horizon_test().unwrap();
+    /// # let transaction_ep   = transaction::All::default().limit(1);
+    /// # let txns             = client.request(transaction_ep).unwrap();
+    /// # let txn              = &txns.records()[0];
+    /// # let account_id       = txn.source_account();
+    /// # let endpoint         = account::Transactions::new(account_id);
+    /// # let first_page       = client.request(endpoint).unwrap();
+    /// # // grab first page and extract cursor
+    /// # let cursor           = first_page.next_cursor();
+    /// let endpoint = account::Payments::new(account_id).cursor(cursor);
+    /// let records  = client.request(endpoint).unwrap();
+    /// #
+    /// # assert!(records.records().len() > 0);
+    /// # assert_ne!(records.next_cursor(), cursor);
+    /// ```
+    pub fn cursor(mut self, cursor: &str) -> Self {
+        self.cursor = Some(cursor.to_string());
+        self
+    }
+
+    /// Fetches all records with a given limit
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// use stellar_client::sync::Client;
+    /// use stellar_client::endpoint::account;
+    ///
+    /// # use stellar_client::endpoint::transaction;
+    /// let client   = Client::horizon_test().unwrap();
+    /// # let transaction_ep   = transaction::All::default().limit(1);
+    /// # let txns             = client.request(transaction_ep).unwrap();
+    /// # let txn              = &txns.records()[0];
+    /// # let account_id       = txn.source_account();
+    /// let endpoint = account::Payments::new(account_id).limit(1);
```

Comment:
Same issue as above with no guarantee that the endpoint could return more than 1 item.